deployment/ccip/changeset/cs_prerequisites.go

```diff
@@ -257,29 +253,6 @@ func deployPrerequisiteContracts(e deployment.Environment, ab deployment.Address
 	} else {
 		lggr.Infow("weth9 already deployed", "addr", weth9Contract.Address)
 	}
-	if linkTokenContract == nil {
```

Comment:
intention is to use the commoncs.DeployLinkToken instead as a common prereq (like MCMS contracts)